cmd/deploy.go

```diff
@@ -275,9 +276,13 @@ func getClusterName(kubeClient *k8s.Client) (string, error) {
 
 func getLatestChart(helmClient *helm.Client, sentryHelmContext *sentry_utils.HelmContext) (*helm.Chart, error) {
 	var err error
-	var chart *helm.Chart
 
-	if chart, err = helmClient.GetLatestChart(CHART_NAME, HELM_REPO_URL); err != nil {
+	if err = helmClient.AddRepo(HELM_REPO_NAME, HELM_REPO_URL); err != nil {
+		return nil, err
+	}
+
+	var chart *helm.Chart
+	if chart, err = helmClient.GetLatestChart(fmt.Sprintf("%s/%s", HELM_REPO_NAME, CHART_NAME)); err != nil {
```

Comment:
fmt.Sprintf("%s/%s", HELM_REPO_NAME, CHART_NAME) is always the same, you can maybe define it as var once